### PR TITLE
OSDOCS-4824_PT2: Additional AWS Quota Updates

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
Enterprise-4.12+ 

Issue:
[OSDOCS-4824](https://issues.redhat.com/browse/OSDOCS-4824)

Link to docs preview:

- [ROSA Quota](https://55495--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-required-aws-service-quotas.html)

QE review:
- [ ] QE has approved this change.

Additional information:
This PR captures some additional changes in the AWS quota section.

### CODE REVIEW

**Review by NautiluX — COMMENTED** [review]
/lgtm

File: modules/rosa-required-aws-service-quotas.adoc [hunk 1]
```diff
@@ -16,16 +16,17 @@ The AWS SDK allows ROSA to check quotas, but the AWS SDK calculation does not in
 
 If you need to modify or increase a specific quota, see Amazon's documentation on link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[requesting a quota increase].
 
+[IMPORTANT]
+====
+For On-Demand Standard (A, C, D, H, I, M, R, T, Z) Amazon EC2 instances, creating a ROSA cluster requires 100 vCPUs or greater for cluster creation. To request for your quota to be increased, open the Service Quotas console within the AWS console.
```
Comment (mburke5678) [inline modules/rosa-required-aws-service-quotas.adoc:21]: > creating a ROSA cluster requires ... for cluster creation

Is this redundant? Just use:

> creating a ROSA cluster requires 100 vCPUs or greater

??

**Comment by mburke5678** [comment]
@EricPonvelle A few nits. Otherwise LGTM

**Pull request merged**